ts/data.ts

```diff
@@ -1,8 +1,43 @@
 /* exported data */
+/* eslint-disable @typescript-eslint/no-unused-vars */
 
-let data = {
+// need an interface for the data object
+// need an interface for the entries (singular Entry object).. what properties should this have?
+interface Entry {
+  image: string;
+  title: string;
+  photoURL: string;
+  note: string;
+  entryId: number;
+}
```

Comment:
You have image and photoURL in your object, but do you really need both? Aren't these the same thing?